packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -383,11 +383,14 @@ <h2 data-translate="pci_database_add_step7_title"></h2>
                                     class="oui-icon oui-icon-component_concept mr-2"
                                 ></span>
                                 <span
-                                    data-translate="pci_database_add_spec_flavor_CPU"
-                                    data-translate-values="{
-                                        cpu: ($ctrl.model.flavor.core|
-                                        bytes:2:false:'GB')
-                                    }"
+                                    data-ng-if="$ctrl.model.flavor.core === 1"
+                                    data-translate="pci_database_add_spec_flavor_CPU_one"
+                                    data-translate-values="{ cpu: $ctrl.model.flavor.core }"
+                                ></span>
+                                <span
+                                    data-ng-if="$ctrl.model.flavor.core > 1"
+                                    data-translate="pci_database_add_spec_flavor_CPU_many"
+                                    data-translate-values="{ cpu: $ctrl.model.flavor.core }"
```

Comment:
**nitpick:** `ng-pluralize` can be used which will simplify the implementation.
For reference, https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html#L229-L237

Reply:
Works perfectly, thanks ! Changed here: https://github.com/ovh/manager/pull/9217/commits/cab6f24a185bc0689d391c90d66773875fd3adc5